### DISCUSSION TITLE
How to print query output rows vertically in duckdb cli?

### DISCUSSION THREAD

**Discussion by liuxhit** [duckdb/duckdb#12549] [Q&A] [answered]
Similar to MySQL, we can display query result vertically by terminating the query with \G instead of a semicolon: https://dev.mysql.com/doc/refman/8.4/en/mysql-tips.html#vertical-query-results

**Comment by szarnyasg** [comment, marked as answer]
Hi @liuxhit, try the [`.columns` dot command](https://duckdb.org/docs/api/cli/dot_commands) or the [`.mode line` output mode](https://duckdb.org/docs/api/cli/output_formats).

**Reply by liuxhit** [reply]
Thank you very much! 
For others who have seen this issue, 1.`.mode line` looks very effective and the default value is `duckbox`, and we can use `.mode` directly to query the output mode in use; 2. use `.rows`(default rendering method) to rollback `.columns`.

**Reply by dossy** [reply]
I've opened up a feature request #17185, for a single-query mechanism to get column-wise (vertical) results for a single query.  If others want this, please upvote it.
